lib/jekyll-github-metadata/repository_finder.rb

```diff
@@ -46,15 +48,19 @@ def nwo_from_config
       end
 
       def git_exe_path
+        exts = (ENV["PATHEXT"] || "").split(File::PATH_SEPARATOR).push("")
```

Comment:
What's going on here?

Reply:
`PATHEXT` on Windows is a list of file extensions respected by cmd.exe.
http://environmentvariables.org/PathExt

This code splits PATHEXT into an array of extensions and adds a blank extension to the end (for odd Windows setups and normal non-Windows setups).

```
irb(main):001:0> (ENV["PATHEXT"] || "").split(File::PATH_SEPARATOR).push("")
=> [".COM", ".EXE", ".BAT", ".CMD", ".VBS", ".VBE", ".JS", ".JSE", ".WSF", ".WSH", ".MSC", ".RB", ".RBW", ""]
```

On non-Windows platforms `exts` is only `[""]`.